<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Academy Engraved LET&quot;;
font: 13pt &quot;Monaco&quot;;</string>
    </property>
    <property name="text">
     <string>   Задание 1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="task1LineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="task2LineEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="task3LineEdit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>190</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Academy Engraved LET&quot;;
font: 13pt &quot;Monaco&quot;;</string>
    </property>
    <property name="text">
     <string>   Задание 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Academy Engraved LET&quot;;
font: 13pt &quot;Monaco&quot;;</string>
    </property>
    <property name="text">
     <string>   Задание 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="answer1Label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>651</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Monaco&quot;;</string>
    </property>
    <property name="text">
     <string> Ответ 1 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="answer2Label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>641</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Monaco&quot;;</string>
    </property>
    <property name="text">
     <string> Ответ 2 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="answer3Label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>651</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Monaco&quot;;</string>
    </property>
    <property name="text">
     <string> Ответ 3 :</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
